tests/unit/orchestrator/search_planner/test_prop_convergence.py

```
@@ -0,0 +1,400 @@
+        planner._phase = "bisect"
+
+        # Drive the planner through its iteration budget.
+        # Return throughput that depends on concurrency to create realistic
+        # bracket updates, but the gap stays too wide to converge.
+        for _ in range(max_iters):
+            if planner.is_converged():
+                break
+            pair = planner.ask()
+            if pair is None:
+                break
+            bench_cfg, variation = pair
```

Ruff (0.15.15)

[warning] Unpacked variable `bench_cfg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)